lib/iris/cube.py

```diff
@@ -484,7 +491,8 @@ def concatenate_cube(self):
         names = [cube.metadata.name() for cube in self]
         unique_names = list(collections.OrderedDict.fromkeys(names))
         if len(unique_names) == 1:
-            res = iris._concatenate.concatenate(self, error_on_mismatch=True)
+            res = iris._concatenate.concatenate(self, error_on_mismatch=True,
+                                                check_aux_coords=check_aux_coords)
```

Comment:
This is line is > 79 characters because pep8 and I never get on. It's not happy with any indentation I try. The best solution I can think of is to assign check_aux_coords to a shorter named variable (at the start of the method) then call that variable here on this line